config/config.go

```diff
@@ -294,47 +294,47 @@ func InitServer() error {
 			configDir = configTmp
 		}
 	}
-	viper.SetDefault("TLSCertificate", filepath.Join(configDir, "certificates", "tls.crt"))
-	viper.SetDefault("TLSKey", filepath.Join(configDir, "certificates", "tls.key"))
-	viper.SetDefault("RobotsTxtFile", filepath.Join(configDir, "robots.txt"))
-	viper.SetDefault("ScitokensConfig", filepath.Join(configDir, "xrootd", "scitokens.cfg"))
-	viper.SetDefault("Authfile", filepath.Join(configDir, "xrootd", "authfile"))
-	viper.SetDefault("MacaroonsKeyFile", filepath.Join(configDir, "macaroons-secret"))
+	viper.SetDefault("Server.TLSCertificate", filepath.Join(configDir, "certificates", "tls.crt"))
+	viper.SetDefault("Server.TLSKey", filepath.Join(configDir, "certificates", "tls.key"))
+	viper.SetDefault("Xrootd.RobotsTxtFile", filepath.Join(configDir, "robots.txt"))
+	viper.SetDefault("Xrootd.ScitokensConfig", filepath.Join(configDir, "xrootd", "scitokens.cfg"))
+	viper.SetDefault("Xrootd.Authfile", filepath.Join(configDir, "xrootd", "authfile"))
+	viper.SetDefault("Xrootd.MacaroonsKeyFile", filepath.Join(configDir, "macaroons-secret"))
 	viper.SetDefault("IssuerKey", filepath.Join(configDir, "issuer.jwk"))
-	viper.SetDefault("OriginUI.PasswordFile", filepath.Join(configDir, "origin-ui-passwd"))
-	viper.SetDefault("OIDC.ClientIDFile", filepath.Join(configDir, "oidc-client-id"))
-	viper.SetDefault("OIDC.ClientSecretFile", filepath.Join(configDir, "oidc-client-secret"))
+	viper.SetDefault("Origin.UIPasswordFile", filepath.Join(configDir, "origin-ui-passwd"))
+	viper.SetDefault("Registry.OIDCClientIDFile", filepath.Join(configDir, "oidc-client-id"))
```

Comment:
Keep this at the `Server` level, not the `Registry` level.

Currently, only the registry uses OIDC ... but I see this for the origin and the director in the near-future as well.